### DISCUSSION TITLE
Install Flowise and also Dify

### DISCUSSION THREAD

**Discussion by HermesMacedo** [coollabsio/coolify#3556] [Feature Requests]
Install Flowise and also Dify.

Both are for building A.I. agents.

https://github.com/FlowiseAI/Flowise
https://github.com/langgenius/dify

**Comment by OhThatMatt** [comment]
There is a open PR for Dify: #3074 

Once it's merged you will be able to use it natively, for now just copy the docker-compose file and try running it that way.

**Comment by peaklabs-dev** [comment]
Dify and Flowise will be coming in one of the next updates via this PR https://github.com/coollabsio/coolify/pull/3838
